modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/macros/Workflow.macro

```diff
@@ -875,20 +875,11 @@ definition {
 			panel = "Applications",
 			portlet = "Process Builder");
 
-		Navigator.gotoNavItem(navItem = "Schemes");
+		Navigator.gotoNavItem(navItem = "Configuration");
 
-		Workflow._tearDownWorkflow();
+		Pagination.changePagination(itemsPerPage = 40);
```

Comment:
Why does the pagination need to be changed?

Reply:
Hello Eric.
I looked in this part of Process Build for a tab with the name "Schemes", it is exactly where the "Configuration" tab is, all tests that use this teardown fail exactly in this part. The teardown clicks on the configuration tab, there it shows the list of components that are or are not configured for the workflow and puts them all to "No workflow", but because it has too many items for a page that only shows 20 items, it fails because the rest of the items are on page 2, so I changed it to the page showing 40 items, which shows all, after that it can change to "no workflow" on all items.
I saw that it failed to test the workflow suite, I'll have to see why.